src/cli/util.js

```diff
@@ -24,6 +24,8 @@ const thirdParty = require("../common/third-party");
 const arrayify = require("../utils/arrayify");
 const isTTY = require("../utils/is-tty");
 
+const requireModule = require("../common/require-module");
```

Comment:
`const requireModule = require;` does this work?

Reply:
Seems to work, yes!

Reply:
I spoke too soon, prod build did not work.

If we do fix this on the prod build, we could remove our custom Babel `transform-custom-require.js` plugin.

I think the main problem is this error:

```
[error] Dynamic requires are not currently supported by @rollup/plugin-commonjs
```

From some quick research I found an idea in <https://github.com/winstonjs/logform/issues/5#issuecomment-467652075> to use what is now [`@rollup/plugin-node-resolve`](https://www.npmjs.com/package/@rollup/plugin-node-resolve) ([here on GitHub](https://github.com/rollup/plugins/tree/master/packages/node-resolve)). Some very interesting reading in these sections:

- https://github.com/rollup/plugins/tree/master/packages/node-resolve#browser
- https://github.com/rollup/plugins/tree/master/packages/node-resolve#using-with-rollupplugin-commonjs
- https://github.com/rollup/plugins/tree/master/packages/node-resolve#resolving-built-ins-like-fs

I am now starting to wonder if we could do the whole prod build with an updated set of Rollup packages, with minimal or even zero Babel transforms, and minimal custom scripting.

Unfortunately I cannot promise when I will have extra time to look into this further. I would be happy if someone else wants to take this over.

Reply:
Right, forgot that. `rollup` replace `require` to this error message.

Reply:
As I said before. `eval('require')` actually a good one.

Reply:
> `eval('require')` actually a good one.

Done but not working. I get this kind of an error on prod test:

```
Cannot find module '../language-html/parser-html' from 'index.js'
```

I tried hacking `scripts/build/babel-plugins/transform-custom-require.js` but with no luck.

The custom Babel plugin assumes that we are using `eval('require')` in every case where we need the transform, which is violated by this proposal.

I will now try another idea.

Reply:
I'm `eval('require')` in the codebase is actually a good solution.